docs/reference/elaborator-reflection.rst

```diff
@@ -32,6 +32,9 @@ Running Elaborator Scripts
 
 On their own, tactics have no effect.
 The meta-operation ``%runElab script`` runs ``script`` in the current elaboration context.
+Before you can use ``%runElab``, you will have to enable the language extension by adding
+``%language ElabReflection`` in your file (or by passing ``-X ElabReflection`` to the
```

Comment:
I think not; with my version of idris (1.0), it's sufficient to just include this line, OR to pass in the flag. I used this program (with an example from @david-christiansen's paper) to test this right now:

```
module Main

import Language.Reflection.Elab

%language ElabReflection

mkId : Elab ()
mkId =
  do x <- gensym "x"
     attack
     intro x
     fill (Var x) ; solve
     solve

idNat : Nat -> Nat
idNat = %runElab mkId

idUnit : () -> ()
idUnit = %runElab mkId

main : IO ()
main = putStrLn (show $ idNat 12)
```